wfms/camunda/v1/WfMSPlugin.java

```diff
@@ -0,0 +1,77 @@
+package cloud.benchflow.plugins.camunda.v1;
```

Comment:
I'll remove the version from the package name.
While we are at it, I think we should formally state the assumptions we make for versions.
We need to have some assumptions on the format of the versions, if we want to recognize version ranges.

At this moment, versions are implemented in a very simple way: either by a fixed range number (`v{Int}`) or range (`v{Int}-v{Int}`).
This does not work very well because for realistic systems versions are not represented in this way; for example, Camunda apparently uses semantic versioning with a `major.minor.patch` format.

Can we assume that all the plugins we will have in BenchFlow will refer to SUTs that implement the SemVer format? If we do, I can adapt the current implementation to recognise SemVer.
Otherwise, we will have to implement a different solution: either implement some sophisticated version recognition system in benchflow, or use fixed versions without ranges.